purchases/src/main/kotlin/com/revenuecat/purchases/Offerings.kt

```diff
@@ -24,4 +25,56 @@ data class Offerings(
      * @param identifier Offering identifier
      */
     operator fun get(identifier: String) = getOffering(identifier)
+
+    /**
+     * Retrieves an specific offering by a placement identifier.
+     * For more info see https://www.revenuecat.com/docs/tools/targeting
+     * @param placementId Placement identifier
+     */
+    fun getCurrentOfferingForPlacement(placementId: String): Offering? {
+        val placements = this.placements ?: run {
+            return null
+        }
+
+        val offeringForPlacement = placements.offeringIdsByPlacement[placementId]?.let { getOffering(it) }
+        val fallbackOffering = placements.fallbackOfferingId?.let { getOffering(it) }
+
+        val showNoOffering = placements.offeringIdsByPlacement.containsKey(placementId)
+        val offering = offeringForPlacement ?: if (showNoOffering) null else fallbackOffering
+
+        return offering?.withPlacement(placementId)
+    }
+}
+
+/**
+ * This class contains all the offerings by placement configured in RevenueCat dashboard.
+ * For more info see https://www.revenuecat.com/docs/targeting
+ * @property fallbackOfferingId The optional offering identifier to fallback on if the placement isn't found.
+ * @property offeringIdsByPlacement Dictionary of all offering identifiers keyed by their placement identifier.
+ */
+private data class Placements(
```

Comment:
I think this should be `internal`, not `private`, since it's used in the `OfferingParser`?